diff --git a/src/protocols/rdp/pointer.c b/src/protocols/rdp/pointer.c
index 7f4901e3..d742045a 100644
--- a/src/protocols/rdp/pointer.c
+++ b/src/protocols/rdp/pointer.c
@@ -47,7 +47,7 @@ BOOL guac_rdp_pointer_new(rdpContext* context, rdpPointer* pointer) {
     cairo_surface_t* surface;
 
     /* Convert to alpha cursor if mask data present */
-    if (pointer->andMaskData && pointer->xorMaskData)
+    //if (pointer->andMaskData && pointer->xorMaskData)
         freerdp_image_copy_from_pointer_data(data,
                 guac_rdp_get_native_pixel_format(TRUE), 0, 0, 0,
                 pointer->width, pointer->height, pointer->xorMaskData,
